README.md

```diff
@@ -359,30 +361,33 @@ will kick off an asynchronous load of the specified media
 playlist. Once it has been retreived, it will become the active media
 playlist.
 
-#### hls.segmentXhrTime
+#### hls.systemBandwidth
 Type: `number`
 
-The number of milliseconds it took to download the last media segment.
-This value is updated after each segment download completes.
+`systemBandwidth` is a combination of two serial processes bit-rates. The first
+is the network bitrate provided by `bandwidth` and the second is the bitrate of
+the entire process after that - decryption, transmuxing, and appending - provided
```

Comment:
May be easier to do a parenthetical

Reply:
`(decryption, transmuxing, and appending)` ?